src/zAssign10InstructorHint/simulateTimeExample.java

```diff
@@ -0,0 +1,53 @@
+package zAssign10InstructorHint;
```

Comment:
Take this out of it's folder in put it in `src`, normally we shouldn't hold example files here for reference. But for this case I'll allow it. Once we don't need this anymore we must trash it